diff --git a/webroot/.htaccess b/webroot/.htaccess
index f43f1c2..ca9e938 100644
--- a/webroot/.htaccess
+++ b/webroot/.htaccess
@@ -1,4 +1,5 @@
     RewriteEngine On
+    RewriteBase /weatherph
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_FILENAME} !-f
     RewriteRule ^(.*)$ index.php?url=$1 [QSA,L]
diff --git a/webroot/index.php b/webroot/index.php
index dc6bd02..9c3fcc2 100644
--- a/webroot/index.php
+++ b/webroot/index.php
@@ -63,7 +63,7 @@
  *
  */
 	if (!defined('WEBROOT_DIR')) {
-		define('WEBROOT_DIR', basename(dirname(__FILE__)));
+		define('WEBROOT_DIR', 'weatherph');
 	}
 	if (!defined('WWW_ROOT')) {
 		define('WWW_ROOT', dirname(__FILE__) . DS);
